### DISCUSSION TITLE
Abracadabra: Pull Requests Wizardry 🧙  [Pull Requests Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#58811] [Pull Requests]
Hocus pocus 🐰! We may not be pulling rabbits out of hats, but Copilot for Pull Requests is just as magical. But first, celebrate our members who have charmed us with their GitHub knowledge and willingness to help others 🍀. 

⚡**Pull Requests Praise**: Thanks to these fine folks for keeping our Community awesome ✔️
* @nicholaspshaw: They had a short and sweet [answer](https://github.com/orgs/community/discussions/57575) to a PR reviewers question which they also [self-solved](https://github.com/orgs/community/discussions/57241) for themselves. Way to go!
* @aahouzi: To the rescue with [search syntax](https://github.com/orgs/community/discussions/58170#discussioncomment-6223643) to find all open PRs. 

Want to see your name above in next month’s check-in? Help us [solve the unsolved](https://github.com/orgs/community/discussions/categories/pull-requests?discussions_q=is%3Aopen+category%3A%22Pull+Requests%22+label%3AQuestion+is%3Aunanswered) with questions like [this one](https://github.com/orgs/community/discussions/56528). 

ℹ️ **What’s new**: The latest Pull Requests updates
* **Fix to improve security around creation of pull requests in public repos** 🔐: We've shipped a small fix to improve security around the creation of pull requests in public repos. [View the changelog](https://github.blog/changelog/2023-05-09-more-secure-private-attachments) and [give feedback](https://github.com/orgs/community/discussions/57972). 
Copilot for Pull Requests: Make your pull requests magical 🪄. Copilot for PRs brings the power of Copilot to the PR experience, to help you write better PR descriptions, and to help your team review and merge PRs faster. [Sign up for the waitlist](https://githubnext.com/projects/copilot-for-pull-requests). 

📣**Announcements**: The latest updates in the GitHub Universe 🆕
* **Happy Pride** 🌈 🏳️‍🌈 🏳️‍⚧️ ❣️: Celebrate Pride with GitHub and the community. [View our post](https://github.com/orgs/community/discussions/58667) on queer open source projects to contribute to and more. 
* **Exploring GitHub with the redesigned navigation, now in public beta**: No sleight of hand here, GitHub’s redesigned navigation is enabled for all users as a public beta.
 [Read on through this link](https://github.blog/2023-06-15-exploring-github-with-the-redesigned-navigation-now-in-public-beta/). 
* **GitHub Availability Report**: May 2023: In May, we experienced four incidents that resulted in degraded performance across GitHub services. [This report](https://github.blog/2023-06-14-github-availability-report-may-2023/) also sheds light on three April incidents that resulted in degraded performance across GitHub services.

What tricks will you get up to with pull requests this month? Share in the comments below.

**Comment by moonfury-ops** [comment]
Thank you

**Comment by queenofcorgis** [comment]
New Pull Requests check-in dropped - [give it a read here](https://github.com/orgs/community/discussions/62082)!
